samples/nrf9160/modem_shell/README.rst

```diff
@@ -0,0 +1,465 @@
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/modem_shell`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+Testing
+=======
+
+To test MoSh after programming the application and all prerequisites to your development kit, complete the following steps:
+
+1. Connect the development kit to the computer using a USB cable.
+   The development kit is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
+
+#. Create a serial connection to the development kit (J-Link COM port) with a terminal using the following settings:
+
+   * Hardware flow control: disabled
+   * Baud rate: 115200
+   * Parity bit: no
+
+#. Reset the development kit.
+
+#. Observe in the terminal window that the application starts.
+   This is indicated by output similar to the following (there is also a lot of additional information about the LTE connection)::
+
+      *** Booting Zephyr OS build v2.4.99-ncs1-3525-g4d068de3f50f  ***
+
+      MOSH version:       v1.5.0-649-g7e657c2fab02
+      MOSH build id:      152
+      MOSH build variant: normal
+
+      Initializing modemlib...
+
+      Initialized modemlib
+      Network registration status: searching
+      Network registration status: Connected - home network
+      mosh:~$
+
+#. Type any of the commands listed in the Overview section to the terminal. When you type only the command, the terminal shows the usage, for example ``sock``.
+
+PPP support
+============
+
+To build the MoSh sample with PPP/dial up support for Windows, use the ``-DOVERLAY_CONFIG=overlay-ppp.conf`` option.
+For example:
+
+``west build -p -b nrf9160dk_nrf9160ns -d build -- -DOVERLAY_CONFIG=overlay-ppp.conf``
+
+See :ref:`cmake_options` for more instructions on how to add this option.
+
+Application FOTA support
+========================
+
+To build the MoSh sample with application FOTA support, use the ``-DOVERLAY_CONFIG=overlay-app_fota.conf`` option.
+For example:
+
+``west build -p -b nrf9160dk_nrf9160ns -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf``
+
+See :ref:`cmake_options` for more instructions on how to add this option.
+
+LwM2M carrier library support
+=============================
+
+To build the MoSh sample with LwM2M carrier library support, use the ``-DOVERLAY_CONFIG=overlay-lwm2m_carrier.conf`` option.
+For example:
+
+``west build -p -b nrf9160dk_nrf9160ns -d build -- -DOVERLAY_CONFIG=overlay-lwm2m_carrier.conf``
+
+See :ref:`cmake_options` for more instructions on how to add this option.
+
+Dependencies
+************
+
+This sample uses the following libraries:
+
+From |NCS|
+  * :ref:`lte_lc_readme`
+  * :ref:`modem_info_readme`
+  * :ref:`at_cmd_readme`
+  * :ref:`at_cmd_parser_readme`
+  * :ref:`at_notif_readme`
+  * :ref:`sms_readme`
+
+From nrfxlib
+  * :ref:`nrfxlib:nrf_modem`
+
+References
+**********
+
+`nRF9160 Hardware Verification Guidelines (nWP-034) <https://infocenter.nordicsemi.com/pdf/nwp_034.pdf>`_
```

Comment:
This works, but if you added the link to the links.txt file, you can just have `` `nRF9160 Hardware Verification Guidelines`_ ``

Reply:
Changed